src/sentry/features/__init__.py

```diff
@@ -205,6 +205,7 @@
 default_manager.add('organizations:team-project-creation-all', OrganizationFeature, FeatureHandlerStrategy.REMOTE)
 default_manager.add('organizations:team-project-creation-all-allowlist', OrganizationFeature, FeatureHandlerStrategy.REMOTE)
 default_manager.add("organizations:team-roles", OrganizationFeature, FeatureHandlerStrategy.INTERNAL)
+default_manager.add("organizations:team-workflow-notifications", OrganizationFeature, FeatureHandlerStrategy.INTERNAL)
```

Comment:
@roggenkemper I guess we don't want to use Flagr?

Reply:
decided to stick with internal for now to be safe